src/main/java/org/junitpioneer/jupiter/CartesianEnumSource.java

```diff
@@ -183,9 +183,15 @@ private static void validatePatterns(CartesianEnumSource enumSource, Set<? exten
 
 	}
 
+	/**
+	 * Containing annotation of repeatable {@code CartesianEnumSource}
```

Comment:
Below we end the sentence with a period (.):

```suggestion
	 * Containing annotation of repeatable {@code CartesianEnumSource}.
```